modules/models/src/main/java/org/eclipse/xpanse/modules/models/view/RegisteredServiceVo.java

```diff
@@ -14,16 +14,19 @@
 import java.util.Date;
 import java.util.UUID;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import org.eclipse.xpanse.modules.models.enums.Category;
 import org.eclipse.xpanse.modules.models.enums.Csp;
 import org.eclipse.xpanse.modules.models.enums.ServiceState;
 import org.eclipse.xpanse.modules.models.resource.Ocl;
+import org.springframework.hateoas.RepresentationModel;
 
 /**
  * Define view object for UI Client query registered service.
  */
+@EqualsAndHashCode(callSuper = true)
 @Data
-public class RegisteredServiceVo {
+public class RegisteredServiceVo extends RepresentationModel<RegisteredServiceVo> {
```

Comment:
Instead of having such a generic list of links, please see if we can have a  specific field called "openApiUrl" and return the URL there?

Reply:
Ok. I will fill the url into field 'openApiUrl'.